modules/telco-ran-workload-partitioning.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-ran-ref-du-components.adoc
+
+:_content-type: REFERENCE
+[id="telco-ran-workload-partitioning_{context}"]
+= Workload partitioning
+
+Description::
+Workload partitioning pins {product-title} platform and Day 2 Operator Pods that are part of the DU profile to the reserved `cpuset` and removes the reserved CPU from node accounting.
+This leaves all unreserved CPU cores available for user workloads.
++
+The method of enabling and configuring Workload partitioning changed in {product-title} 4.13.
++
+--
+4.12 and earlier::
```

Comment:
I would move 4.12 and earlier below 4.13 and later. I also might consider making it a note like 
```suggestion
If you are using version 4.12 and earlier or {product-title,  partitions are set with extra `MachineConfiguration` CRs applied at install-time. ::
```
Since we would want the user to be using the latest version. I recognize we can't remove it because of support, but it might be better to call out the later versions first.